recipes/GeoTessCPP/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "GeoTessCPP" %}
+{% set version = "2.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sandialabs/GeoTessCPP/archive/v{{ version }}.tar.gz
+  sha256: 64a50ea5c40609a7e965a2c33a4aacfff03ffa5de5b94d81bf9c6fab13ef7c59
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - make
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libgeotesscpp$SHLIB_EXT  # [unix]
+    - test -f $PREFIX/lib/libgeotessamplitudecpp$SHLIB_EXT  # [unix]
+
+about:
+  home: https://www.sandia.gov/geotess/ 
+  summary: '3D Earth model parameterization, storage, and interrogation software'
+  description: |
+    A model parameterization and software support system that implements the
+    construction, population, storage and interrogation of data stored in 3D Earth
+    models. This version is in C++ and intended for Linux systems.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

Comment:
This is this package license: https://github.com/sandialabs/GeoTessCPP/blob/master/LICENSE